### ISSUE TITLE
Custom namespaces unavailable

### ISSUE THREAD

**Issue by kghbln** [WikiApiary/WikiApiary#146]
Currently the custom namespaces are going blanc.

**Comment by kghbln** [comment]
Hmm, now all namespaces are gone except for "Special:" (-1)

**Comment by kghbln** [comment]
Everything is fluffy now. Error log will presumable still be worth a check.

**Comment by plexish** [comment]
The site was entirely inaccessible for a while earlier too (between this getting reported and everything working) with blank white pages loading, maybe related.

**Comment by jthingelstad** [comment]
This was an extension update gone awry. I did a composer update and all seemed okay but clearly wasn't (think mediawiki cache may have hidden the issue at first). The good news is both Pingdom and UptimeRobot monitor WikiApiary and triggered alarms. The bad news is I was asleep already. :-) I fixed shortly after waking up. Apologies for the extended outage. I created #147 so this is at least handled better in the future.
